dev/setup-fleet

```diff
@@ -14,11 +14,18 @@ fi
 host=$( docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' k3d-$cluster_name-server-0 )
 ca=$( kubectl config view --flatten -o jsonpath='{.clusters[?(@.name == "k3d-$cluster_name")].cluster.certificate-authority-data}' | base64 -d )
 server="https://$host:6443"
-helm -n cattle-fleet-system upgrade --install --create-namespace --wait fleet-crd charts/fleet-crd
-helm -n cattle-fleet-system upgrade --install --create-namespace --wait --reset-values \
+while !  helm -n cattle-fleet-system upgrade --install --atomic --create-namespace --wait fleet-crd charts/fleet-crd
```

Comment:
Wouldn't this cause to retry indefinitely regardless of the reason of the failure? If the purpose of the loop is to give it enough time for it to deploy, I think we should instead use `--timeout` with a sensible value (defaults to 5 minutes)

Reply:
The purpose is to retry on transient issues, but if it's a permanent one, you're right. Using `--timeout` might not fix that issue, though. So I might need to add a bit more of complexity to it to make it fail, perhaps after three attempts. What do you think?

Reply:
I'd consider that to be a part of the test setup. It shouldn't fail. If it fails we can just ctrl-c it locally. It's just a dev script.

Reply:
It appears I've confused it with `deploy-fleet.sh`, which is used in the Github Action. But `dev/setup-fleet` is not. I'll remove it and leave it out for the time being.